index.html

```diff
@@ -0,0 +1,606 @@
+          <ul class="clients__list">
+            <li class="clients__item client">
+              <a class="client__link" href="#company-1">
+                <svg class="client__icon">
+                  <use href="./images/sprite.svg#logo-1"></use>
+                </svg>
+              </a>
+            </li>
+
+            <li class="clients__item client">
+              <a class="client__link" href="#company-2">
+                <svg class="client__icon">
+                  <use href="./images/sprite.svg#logo-2"></use>
+                </svg>
+              </a>
+            </li>
+
+            <li class="clients__item client">
+              <a class="client__link" href="#company-3">
+                <svg class="client__icon">
+                  <use href="./images/sprite.svg#logo-3"></use>
+                </svg>
+              </a>
+            </li>
+
+            <li class="clients__item client">
+              <a class="client__link" href="#company-4">
+                <svg class="client__icon">
+                  <use href="./images/sprite.svg#logo-4"></use>
+                </svg>
+              </a>
+            </li>
+
+            <li class="clients__item client">
+              <a class="client__link" href="#company-5">
+                <svg class="client__icon">
+                  <use href="./images/sprite.svg#logo-5"></use>
+                </svg>
+              </a>
+            </li>
+
+            <li class="clients__item client">
+              <a class="client__link" href="#company-6">
+                <svg class="client__icon">
+                  <use href="./images/sprite.svg#logo-6"></use>
+                </svg>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__content">
+          <div class="footer__links-contacts">
+            <a class="footer__logo logo logo--footer" href="./index.html">
+              <span class="logo__web">Web</span>Studio
+            </a>
+            <address class="contacts contacts--footer">
+              <a
+                class="contacts__link contacts__link--footer"
+                href="https://maps.app.goo.gl/sZJowedy8QahNWcp7"
+                target="_blank"
+              >
+                м. Київ, пр-т Лесі Українки, 26
+              </a>
+              <a
+                class="contacts__link contacts__link--footer"
+                href="mailto:info@devstudio.com"
+              >
+                info@devstudio.com
+              </a>
+              <a
+                class="contacts__link contacts__link--footer"
+                href="tel:+380961111111"
+              >
+                +38 096 111 11 11
+              </a>
+            </address>
+          </div>
+
+          <div class="footer__social social">
+            <h3 class="social__title footer-title">приєднуйтесь</h3>
```

Comment:
this is not a footer title, this `h3` is related to `social` block, and all styles should be assigned to `social__title`